### ISSUE TITLE
make build-image try to install sentry-cli

### ISSUE THREAD

**Issue by fredvd** [plone/old_plone.org#5]
I'm playing around with the setup to test if I can build a docker image locally.  Docker starts building but when it tries to to a yarn install, it tries to download a rather old sentry-cli version for which no arm64 version exists (yes I'm working on an M1 mac)

#11 55.23 Error: Unable to download sentry-cli binary from https://downloads.sentry-cdn.com/sentry-cli/1.59.0/sentry-cli-Linux-arm64.

I found out this is because several sentry npm modules are dependencies on the volto package, but they are already older and the 1.59 version of @sentri/cli tries to download a binary on yarn install, but doesn't have an arm version on it's CDN so the download from the constructed URL above fails. 

a) is it necessary to enable and install Sentry by default for all Volto installs?  It depends on binary executables being available for different platforms

b) how can I update this sentry version? I tried with updating several of the other @sentry components and I listed "    "@sentry/cli": "1.71.0".  directly in my top level package.json in the Volto project, but yarn sticks to 1.71.1

I suspect this is coming from the created yarn.lock file, so I tried to update the dependencies there according to https://dev.to/ayc0/yarn-lock-how-to-update-it-1fa2

And I've also added a resolutions configuration to the main package.json like this:

```
  "resolutions": {
    "@sentry/cli": "1.71.0"
  }
```

but  `yarn install ; yarn list | grep sentry\/cli `

keeps listing @sentry/cli as a subdependency of webpack-plugin as a dependcy of @plone/volto@14.0.0-alpha.40 with version 1.59.0 :-(

@ericof @tiberiuichim  .  Any generic yarn/npm  config I can use to solve this issue? And is it really necessary to always include sentry in every Volto project by default?

**Comment by tiberiuichim** [comment]
I'm not sure about how to handle that dependency problem. sentry-cli is not in Volto's yarn.lock file.

Maybe also add sentry webpack plugin as a dependency in your project? 

And about sentry... yeah. It's not ideal. https://github.com/plone/volto/issues/2857

**Comment by fredvd** [comment]
So meanwhile I managed to build the container image from the repo. on my local 'host' terminal yarn list still outputs 1.59 for some pinningsin the dep tree, but after adding the resolutions building the Dockerfile into a local image now works.  

I did have to raise docker machine memory limit from 2Gb to 4Gb because yarn build ran out of memory during the docker build.  For reference I added these to the package.json:

```
  "dependencies": {
    "@sentry/node": "6.16.1",
    "@sentry/browser": "6.16.1",
    "@sentry/integrations": "6.16.1",
    "@sentry/webpack-plugin": "1.18.3",
    "@sentry/cli": "1.71.0",
  },
  "resolutions": {
    "@sentry/cli": "1.71.0"
  }
```

Being able to configure these packages to be pulled in or at least be able to look up how they are registered  would be a nice addition.
